app/parse.py

```diff
@@ -1,6 +1,11 @@
 from dataclasses import dataclass
 from enum import Enum
 
+import requests
+from bs4 import BeautifulSoup
+
+URL_MA = "https://mate.academy/en"
```

Comment:
Consider using a more descriptive variable name instead of `URL_MA` to indicate the purpose of the URL.